test/Generics/generic_types.swift

```diff
@@ -237,4 +237,4 @@ struct UnsolvableInheritance2<T : U.A, U : T.A> {}
 // expected-error@-1 {{inheritance from non-protocol, non-class type 'U.A'}}
 // expected-error@-2 {{inheritance from non-protocol, non-class type 'T.A'}}
 
-enum X7<T> where X7.X : G { case X } // expected-error{{'X' is not a member type of 'X7<T>'}}
+enum X7<T> where X7.X : G { case X } // expected-error{{enum element 'X' of type 'X7<T>' is not a type}}
```

Comment:
Heh, now I'm questioning my own phrasing. "of type Z" usually means "which has type Z", but here we're using "which is a member of type Z". We probably should pick something else that's less ambiguous; anything come to mind?